Frontend/src/pages/Voting-Overview.tsx

```diff
@@ -15,20 +20,79 @@ function VotingOverview() {
   // eligible voting count
   // more?
 
-  // useEffect(() => {
-  //   async function loadDAOInfo() {
-  //     if (daoContract && !currentWinningProposal) {
-  //       console.dir(daoContract)
-  //       const winningProposal = await daoContract.getWinningProposal()
-  //       console.log("Current winning proposal:", winningProposal.toString())
-  //       setCurrentWinningProposal(winningProposal.toString())
-  //     }
-  //   }
+  const totalVotes = proposals.reduce((acc, inc) => {
+    acc += inc.voteCount?.toNumber()
+    return acc
+  }, 0)
 
-  //   loadDAOInfo()
-  // }, [daoContract, currentWinningProposal])
+  return isProposalsLoading ? (
+    <Spinner />
+  ) : proposals.length > 0 ? (
+    <ContentWrapper>
+      <Flex flexDir="column">
+        <Heading as="h2">Voting Overview</Heading>
 
-  return <Box>Current winning proposal: {currentWinningProposal}</Box>
+        <Flex mt={10}>
+          <Text>Voting active? </Text>
+          <Text ml={5}>
+            {proposals.some((proposal) => !proposal.active)
+              ? "Finished"
+              : "Ongoing"}
+          </Text>
+        </Flex>
+
+        <Text mt={5}>Total Votes: {totalVotes}</Text>
+
+        <Flex
+          mt={10}
+          alignItems="center"
+          justifyContent="center"
+          w="full"
+          mb={4}
+        >
+          <Grid
+            templateRows="repeat(2, 1fr)"
+            templateColumns="repeat(2, 1fr)"
+            gap={4}
+            height="fit-content"
+            justifyContent="center"
+          >
+            {proposals.map((proposal, idx) => {
+              const { index, voteCount, collectionSize, active } = proposal
+
+              return (
+                <GridItem
+                  key={`proposal-${idx}`}
+                  flexDirection="column"
+                  border="2px"
+                  borderRadius="10px"
+                  p={10}
+                  bg={
+                    winningProposalIndex?.toString() === index?.toString()
+                      ? "green.100"
+                      : ""
+                  }
+                >
+                  <Text>Index: {index?.toString()}</Text>
+                  <Text>VoteCount: {voteCount?.toString()}</Text>
+                  <Text>
+                    Vote %:{" "}
+                    {totalVotes?.toString() === "0"
+                      ? "0"
+                      : voteCount?.toNumber() / totalVotes?.toNumber()}
+                  </Text>
+                  <Text>Collection Size: {collectionSize?.toString()}</Text>
+                  <Text>Is active: {active?.toString()}</Text>
+                </GridItem>
+              )
+            })}
+          </Grid>
+        </Flex>
+      </Flex>
+    </ContentWrapper>
+  ) : (
+    <div>No proposals</div>
+  )
 }
 
 export default VotingOverview
```

Comment:
Perhaps we could improve the styling here? Add `box-shadow` to create 3D impression? Different font colors? Just make it fancier

![image](https://user-images.githubusercontent.com/40721795/186971512-536b263d-8e83-4f95-802a-5eb321c2de7d.png)

Reply:
Yeah ok, is noted, will do it once we have the basic functionality working.

Reply:
![Screenshot 2022-08-26 at 22 27 20](https://user-images.githubusercontent.com/7684178/186985646-7d2a2c81-dc48-4102-939a-45cf85a0513f.png)

Not the most beautiful styling but yeah I guess better than unstyled...

Reply:
Cool